### DISCUSSION TITLE
Student developers pack

### DISCUSSION THREAD

**Discussion by ayyub444** [community/community#57402] [GitHub Education]
### Hi everyone,

![Capture+_2023-06-07-11-13-40](https://github.com/community/community/assets/114808540/18b3b002-202e-4be2-9148-312f4c7fc673)
I have been approved for student developers pack but still didn't get the benefits yet

**Comment by samus-aran** [comment]
Hey @ayyub444  👋

We've updated our application and re-verification process to ensure that GitHub and our partners can offer exceptional, free benefits to students worldwide during their academic journey.

With this update, when you submit your application, you'll have your academic affiliation verified and gain access to Global Campus right away. Your educational benefits will arrive within 72 hours of this verification. However, during busy periods like the start of a new semester, it may take a bit longer.

You’ll get an email notification once your academic benefits are ready, or you can check your status [here](https://education.github.com/discount_requests/pack_application). Meanwhile, as a verified [GitHub Global Campus](https://education.github.com/globalcampus/student) student, you can explore the platform, connect with a local [Campus Expert](https://education.github.com/students/experts), attend events hosted by student leaders and GitHub Education, discover and contribute to student-created repositories from [GitHub Community Exchange](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/about-github-community-exchange) and much more!
